recipes/libpcsclite/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "pcsclite" %}
+{% set version = "1.9.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pcsclite.apdu.fr/files/pcsc-lite-{{ version }}.tar.bz2
+  sha256: cbcc3b34c61f53291cecc0d831423c94d437b188eb2b97b7febc08de1c914e8a
+
+build:
+  skip: True  # [not linux]
+  script:
+    - set -e
+    - set -x
+    - if test -f Makefile; then make distclean; fi
+    - rm -rf *.cache *.m4 config.guess config.log config.status config.sub depcomp ltmain.sh
+    - autoreconf --verbose --install --force
+    - ./configure --prefix=$PREFIX --sbindir=$PREFIX/bin
+    - make install
+    - make distcheck
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('pcsclite') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ cdt('systemd-devel') }}
+    - autoconf-archive
+    - automake
+    - file
```

Comment:
I struggled to understand why this is still not used, but couldn't figure it out, but it seems no harm.
```suggestion
```